lib/src/divider/divider.component.scss

```diff
@@ -0,0 +1,11 @@
+$hc-divider-width: 1px;
+
+.hc-divider {
+    display: block;
+    margin: 0;
```

Comment:
Recommend changing this to `margin: 10px 0;` so we have a little more whitespace between sections

Reply:
I'm using the padding between lists to have whitespace between elements. By setting the margin to 0, I ensure that any element that has a margin defined will be set to 0. This makes the elements more uniform